### PR TITLE
Feature/hab 13 swiftgen integration

### PR DESCRIPTION
**Pull request by VikkiRy**

### CODE REVIEW

**Comment by nikGayko** [comment]
Add everything in Pods and nested folders into git ignore

**Comment by VikkiRy** [comment]
Pod folder added into .gitignore

File: BadHabbit/BadHabbit.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (nikGayko) [inline BadHabbit/BadHabbit.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist:1]: Add xcshareddata to gitignore

File: BadHabbit/BadHabbit/XCAssets+Generated.swift [hunk 2]
```diff
@@ -0,0 +1,105 @@
+// swiftlint:disable all
```
Comment (nikGayko) [inline BadHabbit/BadHabbit/XCAssets+Generated.swift]: Add generated files from SwiftGen to gitignore

File: BadHabbit/swiftgen.yml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  inputs:
+    - SupportFiles/Assets.xcassets
+  outputs:
+    - templateName: swift5
+      params:
+        forceProvidesNamespaces: true
+      output: XCAssets+Generated.swift
+
+
+## Generate constants for your storyboards and XIBs.
```
Comment (nikGayko) [inline BadHabbit/swiftgen.yml]: Clean this file. Commented lines make it's hard to read this config

File: BadHabbit/BadHabbit.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist (hunk 1 above)
Comment (VikkiRy) [reply BadHabbit/BadHabbit.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist:1]: xcshareddata/ added into .gitignore

File: BadHabbit/BadHabbit/XCAssets+Generated.swift (hunk 2 above)
Comment (VikkiRy) [reply BadHabbit/BadHabbit/XCAssets+Generated.swift]: Generated files added into .gitignore

File: BadHabbit/swiftgen.yml (hunk 3 above)
Comment (VikkiRy) [reply BadHabbit/swiftgen.yml]: Unused code removed

**Review by nikGayko — APPROVED** [review]

**Review by HKurylchykava — APPROVED** [review]

**Pull request merged**